gateserver/utils/structparse.py

```diff
@@ -0,0 +1,38 @@
+from collections import namedtuple
+from struct import Struct
+from enum import Enum
+
+STRUCT_FORMAT = '<' # little-endian, no alignment (i.e. packed)
```

Comment:
Pôvodne sa volalo `ENDIANITY`, ale v skutočnosti to definuje aj alignment (or lack thereof). Tak neviem. Ale tiež mám tendenciu vrátiť to.